# HG changeset patch
# Parent 286765d233568af58a49fd0b2c79b63fcabaf9f4
Use /run instead of localstatedir/run/lightdm

Index: lightdm-1.17.3/src/Makefile.am
===================================================================
--- lightdm-1.17.3.orig/src/Makefile.am
+++ lightdm-1.17.3/src/Makefile.am
@@ -83,7 +83,7 @@ lightdm_CFLAGS = \
 	-DSBIN_DIR=\"$(sbindir)\" \
 	-DUSERS_DIR=\"$(localstatedir)/lib/lightdm-data\" \
 	-DLOG_DIR=\"$(localstatedir)/log/lightdm\" \
-	-DRUN_DIR=\"$(localstatedir)/run/lightdm\" \
+	-DRUN_DIR=\"/run/lightdm\" \
 	-DCACHE_DIR=\"$(localstatedir)/cache/lightdm\" \
 	-DSESSIONS_DIR=\"$(pkgdatadir)/sessions:$(datadir)/xsessions:$(datadir)/wayland-sessions\" \
 	-DWAYLAND_SESSIONS_DIR=\"$(datadir)/wayland-sessions\" \
